stylesheets/_session.scss

```diff
@@ -338,7 +340,7 @@ $session_message-container-border-radius: 5px;
     border-radius: 0px;
   }
 
-  &:hover {
+  & > *:not(svg):hover {
```

Comment:
What is this used for?

Reply:
So that when svgs are rendered inside buttons, their opacity is not unnecessarily dimmed from the CSS filter ```brightness```